### ISSUE TITLE
Supporting and extending multi-tenants, auto-update

### ISSUE THREAD

**Issue by weedkiller** [Platformus/Platformus#54]
Hi, can you please also support 

multitenancy, baking this early into platformus will make it easier for hosting
Auto-update from inside the backend admin like wordpress will also be a nice feature.

thanks for the great product!

**Comment by DmitrySikorsky** [comment]
Hi! Thank you for you comment! As I understand we are talking about the multiple sites support in the same backend, right? I'm going to add this feature, but now current version development took more than 1 year, so I want to release version 1 asap. That's why I want to implement this feature in the version 2.

The same about extensions manager and update manager. Underlying ExtCore framework supports adding extensions, so it will be implemented, but a bit later.

**Comment by weedkiller** [comment]
Agreed, and thanks for being receptive!

Its hightime wordpress got some competition and _I see your frame work could succeed_ where orchard failed!  👍 

Wordpress has the concept of multi-site which makes it very easy to maintain, this is an advantage where many larger hosting/advertising companies prefer to maintain one install-instance with many tenants.

Object change notifications:
Also for the RSS part (authors, pages, blogs, posts) if your base content object also supported native _pub-sub_ or [**webhooks**](https://blogs.msdn.microsoft.com/webdev/2016/12/14/introducing-microsoft-asp-net-webhooks-preview-2/), then, it would be easy for a plug-in developer to subscribe to change notifications **at both plug-in level and web API/SOA level**

**Comment by DmitrySikorsky** [comment]
Yes, this is good idea. Also I liked the idea about events (when the object is changed etc,), will do that!

**Comment by weedkiller** [comment]
just use parts from here... its already made for you https://github.com/skoruba/IdentityServer4.Admin

dumb it down just for the features you need, using his options. That way when people download it will run & you can manage users/claims/tennants from the UI he has.
